ci/test/00_setup_env_native_fuzz.sh

```diff
@@ -6,15 +6,15 @@
 
 export LC_ALL=C.UTF-8
 
-export DOCKER_NAME_TAG="ubuntu:20.04"
+export DOCKER_NAME_TAG="debian:buster-backports"
 export CONTAINER_NAME=ci_native_fuzz
 export PACKAGES="clang llvm python3 libevent-dev bsdmainutils libboost-filesystem-dev libboost-test-dev libboost-thread-dev"
 export DEP_OPTS="NO_UPNP=1 DEBUG=1"
 export CPPFLAGS="-DDEBUG_LOCKORDER -DENABLE_DASH_DEBUG -DARENA_DEBUG"
-export CXXFLAGS="-Werror -Wno-unused-command-line-argument -Wno-unused-value"
+export CXXFLAGS="-Werror -Wno-unused-command-line-argument -Wno-unused-value -Wno-deprecated-builtins"
 export PYZMQ=true
 export RUN_UNIT_TESTS=false
 export RUN_INTEGRATION_TESTS=false
 export RUN_FUZZ_TESTS=true
 export GOAL="install"
-export BITCOIN_CONFIG="--enable-glibc-back-compat --enable-zmq --disable-ccache --enable-fuzz --with-sanitizers=fuzzer,address,undefined --enable-c++17 CC=clang CXX=clang++"
+export BITCOIN_CONFIG="--enable-zmq --disable-ccache --enable-fuzz --with-sanitizers=fuzzer,address,undefined --enable-c++17  --enable-suppress-external-warnings CC=clang-15 CXX=clang++-15"
```

Comment:
Those `PACKAGES` are defined because of Bitcoin installing packages per-run instead of using a pre-made image, we might migrate to that in the future to increase our coverage to other Linux distros and keeping them would make backports easier. I think we should keep it in.